IlmBase/IlmThread/IlmThreadMutex.h

```diff
@@ -70,8 +70,12 @@
 #include "IlmBaseConfig.h"
 #include "IlmThreadNamespace.h"
 
+#if (defined(_WIN32) || defined(_WIN64))
```

Comment:
should include a HAVE_PTHREAD guard here?

Reply:
I don't think so, if pthread is not present, the mingw header will just include windows.h and be done.